_posts/2020-09-18-mahdy.git.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: “Welcome to our second slide!”
+---
+Your test
+Use the left arrow to go back!
```

Comment:
---
layout: slide
title: “Welcome to our second slide!”
---
Your test
Use the left arrow to go back.......